internal/config/config.go

```diff
@@ -484,40 +488,31 @@ func (c *Config) GetProvider() cloudprovider.Provider {
 	return cloudprovider.Unknown
 }
 
-// GetMeasurements returns the configured measurements or nil if no provder is set.
-func (c *Config) GetMeasurements() measurements.M {
-	if c.Provider.AWS != nil {
-		return c.Provider.AWS.Measurements
+// GetAttestationConfig returns the configured attestation config.
+func (c *Config) GetAttestationConfig() AttestationCfg {
+	if c.Attestation.AWSNitroTPM != nil {
+		return c.Attestation.AWSNitroTPM
 	}
-	if c.Provider.Azure != nil {
-		return c.Provider.Azure.Measurements
+	if c.Attestation.AzureSEVSNP != nil {
+		return c.Attestation.AzureSEVSNP
 	}
-	if c.Provider.GCP != nil {
-		return c.Provider.GCP.Measurements
+	if c.Attestation.AzureTrustedLaunch != nil {
+		return c.Attestation.AzureTrustedLaunch
 	}
-	if c.Provider.OpenStack != nil {
-		return c.Provider.OpenStack.Measurements
+	if c.Attestation.GCPSEVES != nil {
+		return c.Attestation.GCPSEVES
 	}
-	if c.Provider.QEMU != nil {
-		return c.Provider.QEMU.Measurements
+	if c.Attestation.QEMUVTPM != nil {
+		return c.Attestation.QEMUVTPM
 	}
-	return nil
+	return &DummyCfg{}
```

Comment:
does DummyCfg also handle openstack?

Reply:
OpenStack uses `QEMUVTPM`.
Fixed a case where this was not set correctly.
Will have to check with @malt3 to see if I broke anything OpenStack related